doozer/doozerlib/backend/konflux_olm_bundler.py

```diff
@@ -141,24 +139,65 @@ async def _rebase_dir(
             )
 
         logger = self._logger.getChild(f"[{metadata.distgit_key}]")
-        operator_manifests_dir = operator_dir.joinpath(csv_config['manifests-dir'])
-        operator_bundle_dir = operator_manifests_dir.joinpath(csv_config['bundle-dir'])
+        
+        # Use OLM bundle structure for OADP groups, traditional structure for others
+        use_olm_bundle_structure = self.group.startswith("oadp-")
```

Comment:
```suggestion
        use_olm_bundle_structure = self.group.startswith("oadp-")
```
This may eventually be a longer boolean evaluation expanding to include -- minimally -- ` or ... "mtc-") or ... "mta-")`.